prisma/schema.prisma

```diff
@@ -12,9 +12,93 @@ datasource db {
 }
 
 model Pokemon {
-    id          Int       @id @default(autoincrement())
-    name        String
-    art         String
-    type1       String
-    type2       String?
-}
\ No newline at end of file
+  id             Int              @id @default(autoincrement())
+  name           String
+  art            String
+  type1          String
+  type2          String?
+  ratingAvg      Int              @default(1)
+  ratingCount    Int              @default(0)
+  createdAt      DateTime         @default(now())
+  updatedAt      DateTime         @updatedAt
+  userTeamMember UserTeamMember[]
+  rating         Rating[]
```

Comment:
ratingCount works, but perhaps there is a way to find the count through the number of rating models related to a Pokemon? Achieves the same thing but with one less key.

Reply:
I completely agree with Nathan, here. Well done, Nathan.

Usually you won't store counts of things in a db table directly - it's almost always a calculation done on the fly. You'd display the number of ratings a thing has by counting how many `Rating` records exist for the thing in question.

You can move forward with this path you've chosen if you wish, there will be learnings to be had by doing it... remember to update this ratingCount every time a rating is added or removed :)